src/robot_properties_solo/config.py

```diff
@@ -140,10 +127,10 @@ class Solo8Config(SoloAbstract):
         0,
     ]
 
-    q0 = np.zeros(robot_model.nq)
+    q0 = zero(robot_model.nq)
     q0[:] = initial_configuration
```

Comment:
with the change, this throws an error on my computer
`ValueError: could not broadcast input array from shape (15) into shape (15,1)`
I suggest changing into
```suggestion
    q0 = initial_configuration
    ```
this solved my issue
```

Reply:
This seems to be caused by the change of array convention in pinocchio. Is your pinocchio up-to-date? The problem of the suggested change is that it makes q0 a list instead of a numpy array.

Reply:
I have pinocchio 2.5.2 - is it too old? but good point, my fix is potentially an issue too...

Reply:
That's strange. I have the same version but I don't get this error. When I call pinocchio.utils.zero(n), I get a numpy array of shape (n,) instead of (n, 1), is this the same case with you? Previously, the default shape was (n, 1) but LAAS changed it at a certain point.
At the end of the day, I'd prefer a solution that works with both shapes to not break apart some legacy code written in old pinocchio convention, but maybe that's not necessary.

Reply:
ok - when I start an isolated python session and call pinocchio.utils.zero(n) I get shape(n,)
however in the robot_properties function, I get shape(n,1).
I checked and versions of pinocchio and numpy are the same in both cases but the same commands have different outputs. I do not know why.
Do you see the same thing?